media/transcoder/api/Transcoder.Samples.Tests/ListJobTemplatesTests.cs

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using Xunit;
+
+namespace Transcoder.Samples.Tests
+{
+    [Collection(nameof(TranscoderFixture))]
+    public class ListJobTemplatesTest
+    {
+        private TranscoderFixture _fixture;
+        private readonly CreateJobTemplateSample _createSample;
+        private readonly ListJobTemplatesSample _listSample;
+
+        private string _templateId;
+
+        public ListJobTemplatesTest(TranscoderFixture fixture)
+        {
+            _fixture = fixture;
+            _createSample = new CreateJobTemplateSample();
+            _listSample = new ListJobTemplatesSample();
+            _templateId = $"my-job-template-{_fixture.RandomId()}";
+
+            var result = _createSample.CreateJobTemplate(
+                 _fixture.ProjectId, _fixture.Location,
+                 _templateId);
+
+            Assert.Equal(_fixture.Location, result.JobTemplateName.LocationId);
+            // Job template resource name uses project number for the identifier.
+            Assert.Equal(_fixture.ProjectNumber, result.JobTemplateName.ProjectId);
+            Assert.Equal(_templateId, result.JobTemplateName.JobTemplateId);
+
+            _fixture.JobTemplateIds.Add(_templateId);
+        }
+
+        [Fact]
+        public void ListsJobTemplates()
+        {
+            // Run the sample code.
+            var jobTemplates = _listSample.ListJobTemplates(_fixture.ProjectId, _fixture.Location);
+            string templateName = string.Format("projects/{0}/locations/{1}/jobTemplates/{2}", _fixture.ProjectNumber, _fixture.Location, _templateId);
+            Assert.Contains(templateName, jobTemplates);
```

Comment:
Blocked by error above

Reply:
Almost there, but after changing the ListJobTemplates file, I get the following in ListJobTemplatesTests:

Cannot implicitly convert type 'void' to 'bool' [Transcoder.Samples.Tests]",

Cannot convert lambda expression to intended delegate type because some of the return types in the block are not implicitly convertible to the delegate return type [Transcoder.Samples.Tests]

Reply:
Done